src/libs/store/sagas/refreshContributors.saga.ts

```diff
@@ -5,6 +5,21 @@ import {
   refreshContributorsFailed
 } from 'libs/store/actions';
 import fetchContributors from 'libs/api/fetchContributors';
+import { Contributor, ContributorId } from '../../domain/contributor';
+import { asArray } from '../../utils/env';
+
+const ACCESSIBLE_CONTRIBUTORS_IDS = asArray<ContributorId>(
+  process.env.ACCESSIBLE_CONTRIBUTORS_IDS
+);
+
+const filterAccessibleContributors = (
+  contributors: Contributor[]
+): Contributor[] =>
+  ACCESSIBLE_CONTRIBUTORS_IDS &&
```

Comment:
Please do use the `asArray` env helper method. (or enhance it if the current method isn't sufficient)
> https://github.com/dis-moi/extension/blob/master/src/libs/utils/env.ts

Reply:
You’re right … done